db/migrate/20211102180650_remove_message_bus.rb

```diff
@@ -0,0 +1,16 @@
+class RemoveMessageBus < ActiveRecord::Migration[5.2]
+  def up
+    drop_table :message_bus
+  end
+
+  def down
+    create_table "message_bus", force: :cascade do |t|
+      t.text "channel", null: false
+      t.text "value", null: false
+      t.datetime "added_at", default: -> { "CURRENT_TIMESTAMP" }, null: false
+      t.index ["added_at"], name: "table_added_at_index"
+      t.index ["channel", "id"], name: "table_channel_id_index"
+      t.check_constraint "octet_length(value) >= 2", name: "message_bus_value_check"
+    end
+  end
```

Comment:
I'm not sure we need this `#down` method at all. It can't restore any deleted data anyway. Also we can't use `MessageBus` methods to recreate the table because that class is not present anymore. So I just took a dump of the DDL.

Reply:
Well this is called irreversible migration :) You can look in the code there are some of them ;)
It is totally fine to delete them

Reply:
So what about it? like https://github.com/3scale/porta/blob/3017562449d3275840cae6afe338a01c1dcca899/db/migrate/20100809103633_remove_wrong_cinstance_of_softonic.rb#L8